.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - run:
           name: Run Tests
           command: |
-            bin/dc.sh run --rm --name test-run test-image
+            bin/dc.sh run --name test-run test-image
```

Comment:
Why do we want to stop removing the container here? Does data need to persist between test runs?

Reply:
The next thing that circle does is to `docker cp test-run:/app/test-results $TEST_RESULTS` which requires that the `test-run` container exist. This step gives more info for the test run report in circle. That command was failing due to the missing container causing the test runs to appear to fail.

Reply:
Ah ha! Noted.